addons/dockable_container/split_handle.gd

```diff
@@ -24,16 +24,16 @@ var _dragging = false
 
 func _draw() -> void:
 	var theme_class = SPLIT_THEME_CLASS[layout_split.direction]
-	var icon = get_icon("grabber", theme_class)
-	var autohide = bool(get_constant("autohide", theme_class))
+	var icon = get_theme_icon("grabber", theme_class)
+	var autohide = bool(get_theme_constant("autohide", theme_class))
 	if not icon or (autohide and not _mouse_hovering):
 		return
 	
 	draw_texture(icon, (rect_size - icon.get_size()) * 0.5 )
 
 
 func _gui_input(event: InputEvent) -> void:
-	if event is InputEventMouseButton and event.button_index == BUTTON_LEFT:
+	if event is InputEventMouseButton and event.button_index == MOUSE_BUTTON_LEFT:
 		_dragging = event.is_pressed()
 		if event.doubleclick:
```

Comment:
`InputEventMouseButton.doubleclick` was renamed to `double_click`.
Reference: https://docs.godotengine.org/en/latest/classes/class_inputeventmousebutton.html#class-inputeventmousebutton-property-double-click